### DISCUSSION TITLE
UCB CS126: probability theory - CS自学指南

### DISCUSSION THREAD

**Discussion by giscus** [PKUFlyingPig/cs-self-learning#130] [Announcements]
# UCB CS126: probability theory - CS自学指南

CS自学指南

https://csdiy.wiki/%E6%95%B0%E5%AD%A6%E8%BF%9B%E9%98%B6/CS126/

**Comment by zorwyn** [comment]
这个课程好像没有视频哈，请问是看的notes吗

**Reply by PKUFlyingPig** [reply]
是的

**Reply by pengzhangzhi** [reply]
请问你是已经学过概率论后再重新学一遍还是？

**Comment by Kevin-thu** [comment]
请问有2022spring的答案吗？

**Reply by paraliine** [reply]
同问

**Comment by paraliine** [comment]
请问这本教材的答案有吗？这上面的习题我都想做做试试。

**Comment by MTDickens** [comment]
推荐 MIT : 6.041SC，可以用来代替 cs70 的概率论部分。因为 cs70 的概率论一来有些部分讲的不太清晰，二来有些部分过于简短。

**Comment by Mamihla** [comment]
cs127如何呢，我注意到127的课程主页是开源了recording的

**Reply by rinevard** [reply]
可以参考这个帖子：
https://www.reddit.com/r/berkeley/comments/10734if/eecs_126_or_127/

**Reply by Mamihla** [reply]
127上的差不多了，和帖子里很多评论一样，我也觉得很值得一上

**Reply by liurh299** [reply]
请问cs127的课程主页是这个吗https://www2.eecs.berkeley.edu/Courses/EE127/

**Reply by von-Neumann101** [reply]
感觉CS127概率论偏少了

**Comment by Jinhua-Si** [comment]
您好请问这个课您的github中的notes是按上课顺序排列好的嘛

**Comment by XianglongChen** [comment]
有个疑问，本课介绍编程语言是python，怎么我找了一圈没找到编程作业呢？

**Reply by rinevard** [reply]
在 labs 里可以看到编程作业，里面都是jupyter notebook

**Reply by lildayuan** [reply]
jupyter notebook的中文路径问题请问是怎么解决的呢？我用julia的时候总是自动就下载到c盘去了
然后就有中文路径就用不了，Julia在d盘，jupyter notebook在c盘

**Comment by JimmmmmL** [comment]
这门概率论是偏进阶的，推荐一下如果要入门概率论，可以看哈佛的stat110，我觉得教的也相当不错

**Reply by JimmmmmL** [reply]
https://projects.iq.harvard.edu/stat110/youtube

**Reply by Sunstreamy** [reply]
刚ai给推荐了，就看见你的推荐😄

**Comment by ChinaTTTT** [comment]
考虑到没有视频，我觉得随机过程的部分这门课  ***随机过程（2016年）台湾交通大学陈伯宁*** （https://www.bilibili.com/video/BV12K411K76U/） 能够起到比较好的互补效果，我自己上过觉得老师对数学概念的引出感觉很好，如果可以我就加一下

**Comment by rinevard** [comment]
课本Probability in Electrical Engineering and Computer Science有中译本，名叫EECS应用概率论，译者黄隆波，可以在zlib里找到。

**Reply by tanxy2024** [reply]
求一份电子版的 我在zlib里没找到中文版 谢谢

**Reply by jiyeyuran2899** [reply]
通过网盘分享的文件：EECS应用概率论.pdf
链接: https://pan.baidu.com/s/1CjI3tpdiPOZct-6PeHFXtQ 提取码: 1234

**Comment by ximosss** [comment]
概率论最好的还地是MIT的6.041SC | Fall 2013 | Undergraduate: https://ocw.mit.edu/courses/6-041sc-probabilistic-systems-analysis-and-applied-probability-fall-2013/. 一路follow下来, 真的很轻松, 当然这里我说的轻松指得是logistic上很轻松, content上该不会的还是不会. CS126最新的课程用的教材就是6.041SC的老师写的, 再次让我吹下Tsitsiklis教授, 讲得真好,听得直让人跺脚脚. 集帅(美)们, 直接冲!  一个缺点可能是这门课发布时间在2013年, hw的内容有点不够贴近现代(没有ML方面的内容)

**Reply by tanxy2024** [reply]
问问uu 我看6.041sc是概率论导论是不是只是入门课 没有ucb126讲得深 是学了这个就够了的吗

**Reply by ximosss** [reply]
过了好久...有点忘了. 不过6.041sc应该是那种第一次学概率论看的, 内容很基础(作业不是). ucb126不知道.

**Comment by ximosss** [comment]
最新的是这个: https://eecs127.github.io/index.html 但是已经404
可以用wayback machine找到之前的copy:
https://web.archive.org/web/20240429203928/https://eecs127.github.io/index.html
里面的discussion和homework以及都能下载
视频可以看2022年录的, 是和2024的CS127同一个instructor, YouTube链接:
https://www.youtube.com/playlist?list=PLnocShPlK-Fsj9puVGWJ1byL-WeKW1CMb
其实她的lecture notes已经写的很清楚了, 看完能直接做题的那种:
https://web.archive.org/web/20240429204117/https://eecs127.github.io/assets/notes/eecs127_reader.pdf

Liu Ruiheng ***@***.***> 于2024年6月27日周四 12:23写道：

**Reply by starryious** [reply]
竟然有 wayback machine 这种东西, 现在很多公开课都封闭了, 感觉可以用这个

**Comment by m1a0cX** [comment]
这门课好像也要UCB账号才能看了欸，请问有没有archive的资源或者相同难度平替的课？

**Reply by rinevard** [reply]
我个人觉得 http://prob140.org/ 不错（也是UCB的课），其中的textbook http://prob140.org/textbook 也可以作为学习 CS126 时的参考

**Reply by rinevard** [reply]
关于课程资源，可以参考[PKUFlyingPig的Github库](https://github.com/PKUFlyingPig/EECS126)，这是 2021 Fall 的版本；我上传了[2023 Fall 的版本](https://github.com/rinevard/EECS126)，并做了一个schedule PDF文件。视频似乎从来没有公开过，只能看notes学了。

**Reply by junjiezhou1122** [reply]
我fork一下嘻嘻
